common/nunjucks/src/main/CurioNunjucksEnvironment/index.ts

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
There is no `src/main` pattern in JS, put the tests either as `file.test.js` or `__tests__/file.test.js`

https://stackoverflow.com/questions/62442797/jest-folder-structure